lib/index.js

```diff
@@ -156,6 +156,14 @@
     }
   }
 
+  function respondToPreflight(options, res) {
```

Comment:
I don't think this split out function is needed eith the current changes any longer.

Reply:
Done.